openpype/hosts/houdini/plugins/publish/validate_houdini_license_category.py

```diff
@@ -3,30 +3,29 @@
 from openpype.pipeline import PublishValidationError
 
 
-class ValidateHoudiniCommercialLicense(pyblish.api.InstancePlugin):
-    """Validate the Houdini instance runs a Commercial license.
+class ValidateHoudiniNotApprenticeLicense(pyblish.api.InstancePlugin):
+    """Validate the Houdini instance runs a non Apprentice license.
 
-    When extracting USD files from a non-commercial Houdini license, even with
-    Houdini Indie license, the resulting files will get "scrambled" with
-    a license protection and get a special .usdnc or .usdlc suffix.
+    When extracting USD files from an apprentice Houdini license,
+    the resulting files will get "scrambled" with a license protection
+    and get a special .usdnc suffix.
 
     This currently breaks the Subset/representation pipeline so we disallow
-    any publish with those licenses. Only the commercial license is valid.
+    any publish with apprentice license.
 
     """
 
     order = pyblish.api.ValidatorOrder
     families = ["usd"]
     hosts = ["houdini"]
-    label = "Houdini Commercial License"
+    label = "Houdini Apprentice License"
 
     def process(self, instance):
 
         import hou
 
-        license = hou.licenseCategory()
-        if license != hou.licenseCategoryType.Commercial:
+        if hou.isApprentice():
             raise PublishValidationError(
-                ("USD Publishing requires a full Commercial "
-                 "license. You are on: {}").format(license),
+                ("USD Publishing requires a non apprentice "
+                 "license."),
```

Comment:
I believe these brackets are redundant, right?
```suggestion
                "USD Publishing requires a non apprentice "
                "license.",
```

However, I even think you're not passing the line length either:
```suggestion
                "USD Publishing requires a non apprentice license.",
```